### DISCUSSION TITLE
Custom volume mounts for wordpress service

### DISCUSSION THREAD

**Discussion by kishaningithub** [coollabsio/coolify#2046] [Improvement Requests]
I have a wordpress site that i would like to deploy. Before deploying i would like to make the following volume mount for a configuration file (uploads.ini) in the wordpress container.

```yaml
volumes:
- ./uploads.ini:/usr/local/etc/php/conf.d/uploads.ini # https://www.aptgetlife.co.uk/docker-wordpress-increase-php-max-file-size/
```

The config file (uploads.ini)

```ini
upload_max_filesize = 1G
post_max_size = 1G
```

**Comment by maxxims** [comment]
@kishaningithub  did you manage to resolve this? I am in the same situation.

**Reply by kishaningithub** [reply]
No actually @andrasbacsai any thoughts?

**Reply by maxxims** [reply]
I have added the file in /data and it seems to be picked up by the wordpress container

**Comment by peaklabs-dev** [comment]
@kishaningithub Add the volume mount to the compose file. Or better yet, create a GitHub repo with the file and coolify will automatically get the file from GitHub (fixed in the next release).

**Reply by kishaningithub** [reply]
Is it possible to document this with an example the main documentation site ? It would greatly help coolify users IMO

**Reply by peaklabs-dev** [reply]
Added it to improvements.
**Improve the docs for volume mounts and also the UI in coolify**
